### ISSUE TITLE
Coin and algorithm changes

### ISSUE THREAD

**Issue by VK-VZ** [UnamSanctam/SilentCryptoMiner#2389]
Hello Unam.  I have 2 questions.  
1. I have a cpu mining randomx zephyr and there is a link to the remote pastebin configuration.  Can I change a coin through a link to any other one that works on the randomx algorithm?  
2. And can I change my workers to a different CPU algorithm via a link?
![Screenshot_20240526_134509_Chrome](https://github.com/UnamSanctam/SilentCryptoMiner/assets/149912866/d92b832b-4ee4-462a-bd6f-7a77c2f9f8ef)

**Comment by UnamSanctam** [comment]
1. Yes you just mine to the pool with the other coin, the miner itself only cares about algorithms and not what coin they are used for.
2. Yes the "algo" option is the algorithm: https://github.com/UnamSanctam/SilentCryptoMiner/wiki/CPU-Miner#remote-configuration

**Comment by VK-VZ** [comment]
Thanks.  It turns out that through the pastbin link I can change my workers to any coin and algorithm except gpu?  Or can I change from cpu to gpu too?

**Comment by UnamSanctam** [comment]
Yes you can change everything except the type of miner, because the CPU miner and GPU miner are two different programs.
